packages/core/src/common/utils.ts

```diff
@@ -5,6 +5,14 @@
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
 
+// only accessible within this file, so use `Utils.isNodeEnv` from the outside.
+declare var process: { env: any };
+
+/** Returns whether `process.env.NODE_ENV` exists and equals `env`. */
+export function isNodeEnv(env: string) {
```

Comment:
no, I think it's actually quite meaningful -- why would you compare to other values?

vscode actually does give me autocomplete :P 

![image](https://cloud.githubusercontent.com/assets/723999/23566384/9f62e38a-001f-11e7-94ff-6b3c6153ef4a.png)